apdu-core/src/command.rs

```diff
@@ -78,10 +85,12 @@ impl From<Command> for Vec<u8> {
         } = command;
 
         let mut buffer: Vec<u8> = vec![cla, ins, p1, p2];
+        let mut has_extended_le = false;
         if let Some(mut p) = payload {
             // According to spec, length can be 0, 1 or 2 bytes
             // 2 bytes is prefaced by 00 to differentiate between 1 byte lengths
             if cfg!(feature = "longer_payloads") && p.len() > u8::MAX as usize {
+                has_extended_le = true;
```

Comment:
I think there is no relationship of longer payloads between command and response.
So `has_extended_le` is not needed, isn't it?
I do not know specificly about specification of longer payloads, so it may be my misunderstanding.

Reply:
I realized this was incorrect, but it should be correct now!